.github/workflows/build.yml

```diff
@@ -0,0 +1,57 @@
+name: Build
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  build:
+    name: Build
+    runs-on: windows-latest
+    env:
+      SONAR_SCANNER_VERSION: 4.6.1.2450 # Find the latest version in the "Windows" link on this page:
+                                        # https://sonarcloud.io/documentation/analysis/scan/sonarscanner/
+      SONAR_SERVER_URL: "https://sonarcloud.io"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of analysis
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Set up msbuild
+        uses: microsoft/setup-msbuild@v1.0.2
+      - name: Cache SonarCloud packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.sonar/cache
+          key: ${{ runner.os }}-sonar
+          restore-keys: ${{ runner.os }}-sonar
+      - name: Download and set up sonar-scanner
+        env:
+          SONAR_SCANNER_DOWNLOAD_URL: https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-${{ env.SONAR_SCANNER_VERSION }}-windows.zip
+        run: |
+          New-Item -Force -ItemType directory -Path $HOME/.sonar 
+          curl -sSLo $HOME/.sonar/sonar-scanner.zip ${{ env.SONAR_SCANNER_DOWNLOAD_URL }} 
+          unzip -o $HOME/.sonar/sonar-scanner.zip -d $HOME/.sonar/
+          echo "$HOME/.sonar/sonar-scanner-${{ env.SONAR_SCANNER_VERSION }}-windows/bin" | Out-File -Append -FilePath $env:GITHUB_PATH -Encoding utf8
+      - name: Download and set up build-wrapper
+        env:
+           BUILD_WRAPPER_DOWNLOAD_URL: ${{ env.SONAR_SERVER_URL }}/static/cpp/build-wrapper-win-x86.zip
+        run: |
+          curl -sSLo $HOME/.sonar/build-wrapper.zip ${{ env.BUILD_WRAPPER_DOWNLOAD_URL }} 
+          unzip -o $HOME/.sonar/build-wrapper.zip -d $HOME/.sonar/
+          echo "$HOME/.sonar/build-wrapper-win-x86" | Out-File -Append -FilePath $env:GITHUB_PATH -Encoding utf8
+      - name: Run build-wrapper
+        run: |
+          build-wrapper-win-x86-64 --out-dir build_wrapper_output_directory msbuild sonar_scanner_example.vcxproj 
+      - name: Run sonar-scanner
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          SONAR_ARGS: "--define sonar.projectVersion=1.0 --define sonar.sources=src --define sonar.cfamily.build-wrapper-output=build_wrapper_output_directory --define sonar.sourceEncoding=UTF-8"  
```

Comment:
This applies to all our github actions, something I noticed when doing azure:
You can omit the `--define sonar.cfamily.build-wrapper-output=build_wrapper_output_directory` if you specify it in sonar-project.properties file

Reply:
This needs to be the same dir as passed to build-wrapper, so I think it makes sense to show them in this one file. I think hiding it inside the file (and making things less explicit) will make things worse.
I was considering having global even for this dir.

Reply:
Than we should at lease remove the option from the file. otherwise it is confusing